modules/modelLoader/wuerstchen/WuerstchenModelLoader.py

```diff
@@ -116,7 +116,7 @@ def __load_diffusers(
         elif model_type.is_stable_cascade():
             if prior_prior_model_name:
                 with safe_open(prior_prior_model_name, framework="pt") as f:
-                    if any(key.startswith("down_blocks.0.23") for key in f.keys()):
+                    if any(key.startswith("down_blocks.0.23") for key in f):
```

Comment:
The default iterator in a `dict` is over the keys anyway.

Reply:
This one is a bit dangerous as I don't know if `f` is dict-like.

Reply:
It should be fine, but I will need to check. safe_open returns the list of key/value pairs as a dict

Reply:
Should be fixed by https://github.com/Nerogar/OneTrainer/pull/492/commits/8e30d4e21e1c076f4269d9d57beb7f12603d66b6.